src/test/scala/rx/lang/scala/ConstructorTest.scala

```diff
@@ -32,4 +37,86 @@ class ConstructorTest extends JUnitSuite {
     assertEquals(List(1,2,3), xs)
 
   }
+
+  @Test def testFromEmitter(): Unit = {
```

Comment:
Not sure if `ConstructorTests` are the right place for these, can also move them to a new file